<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>685</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>735</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Isolines</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../build-Isolines-Desktop_Qt_5_12_1_MinGW_64_bit-Release/images/images.qrc">
    <normaloff>:/new/rec/pencil.png</normaloff>:/new/rec/pencil.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>580</width>
     <height>660</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>580</width>
     <height>660</height>
    </size>
   </property>
   <widget class="BackgroundWidget" name="background_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>580</width>
      <height>610</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>580</width>
      <height>610</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>580</width>
      <height>610</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>520</width>
      <height>600</height>
     </size>
    </property>
    <widget class="LegendWidget" name="legend_widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>580</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>580</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>580</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="IsolineWidget" name="isoline_widget" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>522</width>
       <height>500</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAuthor"/>
   </widget>
   <widget class="QMenu" name="menuMods">
    <property name="title">
     <string>Mods</string>
    </property>
    <addaction name="actionShow_isolines"/>
    <addaction name="actionColor_Map_Interpolation"/>
    <addaction name="actionShow_grid"/>
    <addaction name="actionInteractive_isoline_draw"/>
    <addaction name="actionShow_intersection_points"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMods"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>16777215</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionShow_grid"/>
   <addaction name="actionInteractive_isoline_draw"/>
   <addaction name="actionShow_isolines"/>
   <addaction name="actionShow_intersection_points"/>
   <addaction name="actionColor_Map_Interpolation"/>
   <addaction name="actionAuthor"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../build-Isolines-Desktop_Qt_5_12_1_MinGW_64_bit-Release/images/images.qrc">
     <normaloff>:/new/rec/open.png</normaloff>:/new/rec/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="icon">
    <iconset resource="../build-Isolines-Desktop_Qt_5_12_1_MinGW_64_bit-Release/images/images.qrc">
     <normaloff>:/new/rec/question.png</normaloff>:/new/rec/question.png</iconset>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
   <property name="statusTip">
    <string>About author</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../build-Isolines-Desktop_Qt_5_12_1_MinGW_64_bit-Release/images/images.qrc">
     <normaloff>:/new/rec/settings.png</normaloff>:/new/rec/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="statusTip">
    <string>Change settings</string>
   </property>
  </action>
  <action name="actionShow_isolines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../build-Isolines-Desktop_Qt_5_12_1_MinGW_64_bit-Release/images/images.qrc">
     <normaloff>:/new/rec/pencil.png</normaloff>:/new/rec/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Show isolines</string>
   </property>
   <property name="statusTip">
    <string>Show isolines</string>
   </property>
  </action>
  <action name="actionColor_Map_Interpolation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../build-Isolines-Desktop_Qt_5_12_1_MinGW_64_bit-Release/images/images.qrc">
     <normaloff>:/new/rec/paint.png</normaloff>:/new/rec/paint.png</iconset>
   </property>
   <property name="text">
    <string>Color Map / Interpolation</string>
   </property>
   <property name="statusTip">
    <string>Color Map / Interpolation</string>
   </property>
  </action>
  <action name="actionShow_grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../build-Isolines-Desktop_Qt_5_12_1_MinGW_64_bit-Release/images/images.qrc">
     <normaloff>:/new/rec/grid.png</normaloff>:/new/rec/grid.png</iconset>
   </property>
   <property name="text">
    <string>Show grid</string>
   </property>
   <property name="statusTip">
    <string>Show grid</string>
   </property>
  </action>
  <action name="actionInteractive_isoline_draw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../build-Isolines-Desktop_Qt_5_12_1_MinGW_64_bit-Release/images/images.qrc">
     <normaloff>:/new/rec/hand.png</normaloff>:/new/rec/hand.png</iconset>
   </property>
   <property name="text">
    <string>Interactive</string>
   </property>
   <property name="toolTip">
    <string>Interactive isoline drawing</string>
   </property>
   <property name="statusTip">
    <string>Interactive isoline drawing</string>
   </property>
  </action>
  <action name="actionShow_intersection_points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../build-Isolines-Desktop_Qt_5_12_1_MinGW_64_bit-Release/images/images.qrc">
     <normaloff>:/new/rec/drawing-symbol-of-lines.png</normaloff>:/new/rec/drawing-symbol-of-lines.png</iconset>
   </property>
   <property name="text">
    <string>Show intersection points</string>
   </property>
   <property name="statusTip">
    <string>Show intersection points</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BackgroundWidget</class>
   <extends>QWidget</extends>
   <header>backgroundwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LegendWidget</class>
   <extends>QWidget</extends>
   <header>legendwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IsolineWidget</class>
   <extends>QWidget</extends>
   <header>isolinewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../build-Isolines-Desktop_Qt_5_12_1_MinGW_64_bit-Release/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
